src/components/pages/home/index.tsx

```diff
@@ -45,6 +45,9 @@ const Home: FunctionComponent = () => {
   const accessibleReactApps: any = find(homepageData, {
     id: 'accessibleReactApps',
   })
+  const nextjsVideoApp: any = find(homepageData, {
```

Comment:
this variable could be renamed to where it's represented on the page.

`projectFeatureCardVideoApp`